### ISSUE TITLE
Staging Review finding: Form inputs not programmatically associated with labeling element

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#105696]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Education Data Migration
**Product:** WEAMS Public
**Feature:** WEAMS Public

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/licenses-certifications-and-prep-courses/results](https://staging.va.gov/education/gi-bill-comparison-tool/licenses-certifications-and-prep-courses/results?name=&state=all&initial=all&page=1&category=all&category=license&category=certification&category=prep%20course) in the filter accordion, neither "Category type" nor "State" are programmatically associated with the inputs they describe.

When a screen reader user tabs to the first input under "Category type," you hear the label for the checkbox but you don't hear the words "Category type," which is important context for what you're filtering by. Likewise, when you tab to the "State" select input, you hear "Applies to only license and prep course category type. Certifications are available nationwide." but you don't hear "State," which is the actual thing you're filtering by.

<details><summary>Link, screenshot or steps to recreate</summary>

![Filter accordion with Category type and State filters.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/90889-Brian-1.png?raw=true)

</details>

## Recommended action

For the "Category type" filter, use the [label header variant of the va-checkbox-group component](https://design.va.gov/storybook/?path=/story/uswds-va-checkbox-group--label-header).

For the "State" filter, use something like:
```
<fieldset>
   <legend><h3>State</h3></legend>
   <va-select ... />
</fieldset>
```

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jkdrahota** [comment]
<p>This issue has been addressed and testing passed.</p>
